ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+
+### User Onboarding
+
+1. User onboarding happens with the sending anchor, and is out of the scope of this spec.
+1. Sending anchor should use this to collect any KYC about the receiver it will need so it doesn't have to do it at every transaction.
+
+### Payment Flow
+
+1. Payment initiation happens in the sending anchors interface, and is also out of the scope of this spec.
+1. The sending client chooses a destination region.
+1. The sending anchor fetches the TOML file to find the SEND_SERVER API and [`/info`](#info) endpoint fields for the receiving anchor for this region.
+1. Using the fields fetched from /info, the sending anchor collects all needed information from the sending client.
+1. The sending anchor POSTs all that information to the [`/send](#send) endpoint of the receiving anchor.
+1. If information is missing, or the receiving anchor needs extra information to complete this transaction, `POST /send` will return `403 Customer Information Needed` along with the missing fields. The sending anchor should collect these and retry the entire `POST /send` call, until it succeeds.
+1. Once all needed information is provided and the receiving anchor is ready to complete this transaction, the `POST /send` call will return a `200` status along with information needed to complete the transaction over the stellar network.
+1. The sending anchor should collect the fiat from the sending client, and perform the specified stellar transaction to send the money to the receiving anchor.  This is usually a path payment, but can be done with a regular payment as well, so long as the receiving anchor gets the token they expect.
+1. Once the stellar transaction is completed, the receiving anchor should deposit the money in the receivers bank account.
+1. The sending anchor can query the status of this transaction via the [`/transaction`](#transaction) endpoint, and should communicate updates to the sending client as that progresses.
+
+
+
+## API Endpoints
+
+* [`GET /info`](#info): required
+* [`POST /send`](#send): required
+* [`GET /transactions`](#transaction-history): required
+* [`GET /transaction`](#single-historical-transaction): required
+
+### Info
+#### Request
+
+```
+GET DIRECT_PAYMENT_SERVER/info
+```
+
+Allows an anchor to communicate basic info about what currencies their `DIRECT_PAYMENT_SERVER` supports receiving from partner anchors.
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `description` fields in the response should be in this language.
+
+#### Response
+
+The response should be a JSON object like:
+
+```json
+{
+   "receive":{
+      "USD":{
+         "enabled":true,
+         "fee_fixed":5,
+         "fee_percent":1,
+         "min_amount":0.1,
+         "max_amount":1000,
+         "fields":{
```

Comment:
If all fields listed are from a SEP, there's no reason to have a description for each field. Anchors can keep a list of descriptions in their native language instead of relying on the other anchor to return a translation of the their own description. 

Then, the value for `sender`, `receiver`, and `transaction` could be a list of SEP9 field names.

Reply:
I don't think we can require that they need to be from a SEP.  Anchors can request whatever they want.